### ISSUE TITLE
Installation of via Web Installer not possible for ESP32-S2

### ISSUE THREAD

**Issue by franden** [xperiments/xhost-pro#10]
I tried to update my ESP32-S2 with the newst xhost-pro release (0.9.7 ?) today 2022-09-13 via web installer. After few seconds after I push the INSTALL button, I get the message that installation failed. 

![2022-09-13_XHOST-PRO-issue](https://user-images.githubusercontent.com/20822716/189990827-83211df5-37fd-4467-9f43-2e1ad77284eb.gif)

If I ty to push BOOT button before I click on INSTALL button I get the same issue.

**Steps to reproduce:**
1. connect ESP32-S2
2. push and hold both buttons (RESET and BOOT) for few seconds
3. open web installer and click connect
4. select ESP32-S2 device
5. select install XHOST
6. push install button

**Expected behaviour:**
ESP32-S2 must be erased, XHOST-PRO must be installed

**Additonal infromation:**
Last time I installed the XHOST-PRO in spring 2022 ([Release 0.9.5](https://github.com/xperiments/xhost-pro/releases/tag/0.9.5)) via web installer without any issues.

please fix the issue

**Comment by Euphoriarules** [comment]
Got same problem (Windows 11 and usb3 notebook) im try on other computer (also Windows 11) with usb2 port and is working for me.

**Comment by thefunkygibbon** [comment]
yeah it didnt work for me either.  I ended up flashing the firmware file (the main big one)  via https://web.esphome.io/?dashboard_wizard
worked fine.

**Comment by damo503** [comment]
I have a ESP32-S2 Mini V1.0 and I tested it  and it's not working. The problem seems in the firmwares 0.96 and 0.97 and not in the filesystem because the 0.95 firmware with the last filesystem works fine with the esp32-s2 V1.

You can do it  flashing  the firmware 0.95 with a flash-tool or with the url https://web.esphome.io/?dashboard_wizard , and donwload the last esp32-s2-base.fs.ota to have the last goldhen 

to flash the filesystem Ota file  just connect your pc with the xhost-pro 0.95 by wifi (like in your ps4) and  go to 6.6.6.6 url and you have to use the upgrade filesystem option it's just in  Xhost-->Settings-->Xhost-Pro Setup -->firmware -->Flash new filesystem, select your ota file, when it's done,  reset  your Esp32 and enjoy it.

**Comment by welsontec** [comment]
I tested two esp8266 that I have, both tested with ide arduino, cable and usb works perfect. When I install via the web it doesn't work and 1 of these installs with "NodeMCU-PyFlasher-with-ResetLocale" and "esp8266.bin".
What you installed connected and configured on ps4 only appears error screen "Failed to trigger explode..."
I already modified usb speed, but I don't know what else to do. Any tips? Thanks
